src/standalone/Vegetation/canopy_boundary_fluxes.jl

```diff
@@ -334,15 +335,24 @@ function canopy_turbulent_fluxes_at_a_point(
     T_in::FT = Thermodynamics.air_temperature(thermo_params, ts_in)
     ΔT = T_in - T_sfc
     r_ae::FT = 1 / (conditions.Ch * SurfaceFluxes.windspeed(sc))
-    ρ_air::FT = Thermodynamics.air_density(thermo_params, ts_in)
+    ρ_sfc::FT = Thermodynamics.air_density(thermo_params, ts_sfc)
```

Comment:
renaming since we use the surface density, not the atmos density at the first interior point